pandas/tests/series/test_apply.py

```diff
@@ -124,7 +124,7 @@ def test_apply_datetimetz(self):
 
         # change dtype
         result = s.apply(lambda x: x.hour)
-        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int32)
+        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int64)
```

Comment:
It's true it previously gave int32, but is there any reason for that? We almost always use int64 as the default integer size, and also currently Timestamp.hour gives you back int64 when you replace the `map` with an explicit loop:

```
In [20]: dtidx = pd.date_range(start='2012-01-01', periods=4)

In [29]: dtidx.map(lambda x: x.hour)
Out[29]: array([0, 0, 0, 0], dtype=int32)

In [30]: np.array([x.hour for x in dtidx])
Out[30]: array([0, 0, 0, 0])

In [31]: np.array([x.hour for x in dtidx]).dtype
Out[31]: dtype('int64')
```